tools/msconvert/msconvert_macros.xml

```diff
@@ -379,6 +422,37 @@
       </param>
     </section>
 
+    <section name="general_options" title="General Options">
+      <param argument="--combineIonMobilitySpectra" type="boolean" label="Combine ion mobility spectra" help="When false, each mobility scan is written as a separate spectrum. When true, each retention time point will have a single merged scan. For Bruker TIMS spectra, the ion mobilities will be preserved in a separate binaryDataArray, and for TIMS PASEF MS2s, each precursor will be merged separately." />
+
+      <conditional name="scan_summing">
+        <param name="do_scan_summing" type="boolean" truevalue="true" falsevalue="false" label="Sum adjacent scans" help="Sums MS2 sub-scans whose precursors are similar in the m/z, scan time, and/or ion mobility dimensions. It is useful for some Waters DDA data and Bruker PASEF data, where sub-scans should be summed together to increase the SNR" />
+        <when value="false" />
+        <when value="true">
+          <param name="precursorTol" type="float" label="Precursor m/z tolerance" value="0.05" min="0" optional="true" help="Spectra with precursor m/z values with a difference less than this tolerance are summed together." />
+          <param name="scanTimeTol" type="float" label="Scan time tolerance" value="10.0" min="0" optional="true" help="Spectra with scan times with a difference less than this tolerance (in seconds) are summed together." />
+          <param name="ionMobilityTol" type="float" label="Ion mobility tolerance" value="0.01" min="0" optional="true" help="Spectra with ion mobility values with a difference less than this tolerance are summed together. Only relevant for ion mobility spectra." />
+        </when>
+      </conditional>
+
+      <param argument="--simAsSpectra" type="boolean" label="SIM as Spectra" help="Write selected ion monitoring as spectra, not chromatograms" />
+      <param argument="--srmAsSpectra" type="boolean" label="SRM as Spectra" help="Write selected reaction monitoring as spectra, not chromatograms" />
+      <param argument="--acceptZeroLengthSpectra" type="boolean" label="Accept zero-length spectra" help="Some vendor readers have an efficient way of filtering out empty spectra, but it takes more time to open the file" />
+      <param argument="--ignoreUnknownInstrumentError" type="boolean" label="Ignore unknown instrument error" help="If true, if an instrument cannot be determined from a vendor file, it will not be an error" />
+
+      <conditional name="multi_run_output">
+        <param name="do_multi_run_output" type="boolean" truevalue="true" falsevalue="false" label="Output multiple runs per file" help="Some input types can store multiple runs (samples) in a single file (e.g. WIFF). Each run must be written to a separate output file, so check this option if you want to output all runs for a file (each file will create a dataset collection)" />
```

Comment:
IUC guidelines state to use a select-param in conditionals.

Reply:
Yes, and in general I still disagree with that guideline. :) Seeing "no" and "yes" (or a repeat of the label) in a select box is jarring to me. Much more so than options appearing/disappearing/changing based on the yes/no slider.

But reviewing all the current msconvert options there's at least one place where it would make sense to switch from boolean to something else: `Filter MS levels`. I think that would look better as a repeat, like the adjacent `Filter Scan Indices` and "Filter Scan Numbers`.

Reply:
;) you can start discussing this with the IUC. But we could also not use Yes/No in the select, but maybe some short sentence.